test/Test.juvix

```diff
@@ -137,107 +137,86 @@ prop-transposeMatrixDimensions : List (List Int) -> Bool
 sortTest : String -> (List Int -> List Int) -> QC.Test
   | sortName sort :=
     QC.mkTest
-      (QC.testableFunction QC.argumentListInt QC.testableBool)
       ("sort properties: " ++str sortName)
       (prop-sort sort);
 
 dropTest : QC.Test :=
   QC.mkTest
-    (QC.testableFunction
-      QC.argumentNat
-      (QC.testableFunction QC.argumentListInt QC.testableBool))
     "drop properties"
     prop-drop;
 
 snocTest : QC.Test :=
   QC.mkTest
-    (QC.testableFunction
-      QC.argumentListInt
-      (QC.testableFunction QC.argumentInt QC.testableBool))
     "snoc properties"
     prop-snoc;
 
 zipTest : QC.Test :=
   QC.mkTest
-    QC.testableListIntListInt
     "zip properties"
     prop-zip;
```

Comment:
All the properties need to be wrapped in the `Fun` type, see https://github.com/anoma/juvix-quickcheck/blob/0a4cae9daba9e495f1bec00ddec8649c37f25290/Example.juvix#L23

Reply:
Done